components/scream/cime_config/buildnml

```diff
@@ -3,19 +3,27 @@
 """
 Namelist creator for E3SM's SCREAM component
 
-This script is mostly for processing the scream_input.yaml file.
+This scripts takes the namelist_defaults_scream.xml file from the repo
+and produces the processed $case/namelist_scream.xml. From the processed
+file, raw input files are produced which should not be modified by users.
```

Comment:
Maybe add a comment here saying that users can modify $case/namelist_scream.xml? Upon first read, it sounded like defaults-> case xml file -> raw input files without any opportunity for users to modify defaults. Also, it might be easier for readers to understand if "raw input files" was "scream_input.yaml and namelist.nml"... though I get that you're trying to avoid comments which will become wrong when we modify the code to read in xmls.